src/transfers/MultiAppOkapiMooseTransfer.C

```diff
@@ -14,8 +14,9 @@
 #include "libmesh/meshfree_interpolation.h"
 #include "libmesh/system.h"
 
-template<>
-InputParameters validParams<MultiAppOkapiMooseTransfer>()
+template <>
+InputParameters
+validParams<MultiAppOkapiMooseTransfer>()
```

Comment:
I don't want to assume that `Okapi` will always be the master application, but I definitely think that we can combine those transfers into one class. I'll see about working on that, and adding a new test

Reply:
Actually, thinking about this more, the only way the `OkapiMoose` transfer would be useful (at least in my limited brain) was if it was registered with the master (non-`Okapi`) application. Otherwise we'd get an error about a non-registered object when running `other_app-opt -i master.i`. So IMO I think we actually can delete that object...

Reply:
Ohh, that makes sense, I agree we can delete this object